docs/framework/guides/contributing/plugin-metadata.md

```diff
@@ -0,0 +1,33 @@
+---
+category: framework-contributing
+order: 40
+---
+
+# Plugin metadata
+
+Most of CKEditor 5 packages contain a separate `ckeditor5-metadata.json` file in their root directory. This file contains the basic information about plugins that are exported by the package. Only the main ones that put the given functionality together are described in `ckeditor5-metadata.json`. For example, the `ckeditor5-metadata.json` in the `ckeditor5-mention` package does not contain the `MentionEditing` and `MentionUI` plugins, but instead it only contains the metadata for the main `Mention` plugin, which is a "glue plugin", that loads the editing and UI features.
+
+## Data format for plugin metadata
+
+The `ckeditor5-metadata.json` file is a JavaScript object that holds a `plugins` array. Each element of this array is an object containing information about a different plugin  delivered by the package. Below is a list of all properties that can be used to define the metadata for a plugin:
+
+* `name` &ndash; A human-readable name of the plugin.
+* `description` &ndash; A human-readable short description of the plugin.
+* `docs` &ndash; An absolute or relative URL for the plugin's documentation. If this URL is relative, it leads to the CKEditor 5 documentation in [https://ckeditor.com/docs/ckeditor5/latest/](https://ckeditor.com/docs/ckeditor5/latest/).
+* `path` &ndash; A path to the file, relative to the metadata file that exports the plugin.
+* `className` &ndash; The name of the class used to create the plugin.
+* `requires`&ndash; An array of the plugin's soft requirements and other non-explicit requirements.
+* `registeredToolbars`&ndash; An array of all toolbar names registered by the plugin. These names need to contain the full configuration path (e.g. `table.contentToolbar` and `table.tableToolbar` for `Table` plugin).
+* `uiComponents` &ndash; An array of objects, that describe UI components exported by the plugin. Each object in this array may contain:
+	* `name` &ndash; A name of the component the plugin exports.
+	* `type` &ndash; Component type: `Button`, `SplitButton` or `Dropdown`.
+	* `iconPath` &ndash; A path to the SVG icon for `Button` or `SplitButton` components.
+	* `label` &ndash; Text content for `Dropdown` components.
+	* `toolbars`&ndash; An array of toolbar names, a given UI component can be added to. Some features may be added to multiple toolbars.
```

Comment:
There are some missing spaces before &ndash; in this file.

Reply:
Fixed.